recipes/conversation_view/feed_item_row/feed_item_row_default.story.vue

```diff
@@ -18,69 +18,142 @@
-          element-type="button"
-          size="200"
-        >
-          <span v-html="reactions" /> :smile:
-        </dt-emoji-text-wrapper>
+        <dt-recipe-emoji-row
+          :reactions="mockReactions"
+        />
       </template>
-      <template
-        v-if="menu"
-        #menu
-      >
-        <dt-icon :name="menu" />
+      <template #menu>
+        <!-- TODO replace this with DT menu -->
+        <div
+          class="d-d-flex d-bgc-primary d-bar-pill d-bc-default d-ba"
+          role="group"
+        >
+          <dt-button
+            v-for="button of hoverButtons"
+            :key="button"
+            kind="muted"
+            importance="clear"
+            size="xs"
+            aria-label="button"
+          >
+            <template #icon>
+              <slot name="icon">
+                <dt-icon
+                  :name="button"
+                  size="300"
+                />
+              </slot>
+            </template>
+          </dt-button>
+        </div>
       </template>
     </dt-recipe-feed-item-row>
   </ul>
 </template>
 
 <script>
 import DtRecipeFeedItemRow from './feed_item_row.vue';
-import { DtEmojiTextWrapper } from '@/components/emoji_text_wrapper';
+import { DtRecipeEmojiRow } from '../emoji_row';
 import { DtAvatar } from '@/components/avatar';
 import { DtIcon } from '@/components/icon';
+import { DtButton } from '@/components/button';
+import { DtStack } from '@/components/stack';
 
 export default {
   name: 'DtRecipeFeedItemRowDefault',
   components: {
     DtAvatar,
     DtRecipeFeedItemRow,
-    DtEmojiTextWrapper,
+    DtRecipeEmojiRow,
     DtIcon,
+    DtButton,
+    DtStack,
+  },
+
+  data () {
+    return {
+      mockReactions: [
+        {
+          emojiUnicodeOrShortname: '😀',
+          isSelected: true,
+          ariaLabel: 'Emoji aria label',
+          tooltip: 'You reacted with 😀',
+          num: 1,
+        },
+        {
+          emojiUnicodeOrShortname: '😒',
+          isSelected: false,
+          ariaLabel: 'Emoji aria label',
+          tooltip: 'You reacted with 😒',
+          num: 1,
+        },
+        {
+          emojiUnicodeOrShortname: '🙃',
+          isSelected: false,
+          ariaLabel: 'Emoji aria label',
+          tooltip: 'You reacted with 🙃',
+          num: 99,
+        },
+      ],
+
+      hoverButtons: ['bell', 'living-thing', 'map-pin'],
+
+      persons: ['Jim Halpert', 'Michael Scott', 'Pam'],
+    };
   },
 };
 </script>
+
+<style lang="less">
+  .feed-item-row__default_story {
+    p {
+      color: var(--dt-color-foreground-primary);
+      font-size: 15px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 24px;
+    }
+    &__thread{
+      height: 32px;
+      &__reply{
+        color: var(--dt-color-foreground-tertiary);
+        font-size: 12px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 16px;
+        letter-spacing: -0.12px;
+      }
+    }
+  }
```

Comment:
Just a BEM naming issue here. Never go multiple levels deep with double underscore as it creates too much complexity as shown in the BEM FAQ https://getbem.com/faq/#css-nested-elements

This is not that big of deal since it's in a story.vue but we still should follow best practices.

so naming on these would be:

- feed-item-row__default-story
- feed-item-row__thread
- feed-item-row__reply